app.py

```diff
@@ -88,6 +103,41 @@ def events(account_id):
     return Response(None, status=204)
 
 
+@app.route("/v2/<account_id>/tags", methods=["GET", "POST"])
+def tags(account_id):
+    if request.method == "GET":
+        return {
+            "tags" : dripData.tags
+        }
+    elif request.method == "POST":
+        tags_data = request.json.get("tags", [])
+        if len(tags_data) > 0:
+            email = tags_data[0].get("email")
+            new_tag = tags_data[0].get("tag")
+            subscriber_id = dripData.email_id_map.get(email)
+            if subscriber_id:
+                subscriber = dripData.subscribers[subscriber_id]
+                subscriber_tags = subscriber["tags"]
+                if new_tag not in subscriber_tags:
+                    subscriber_tags.append(new_tag)
+                if new_tag not in dripData.tags:
+                    dripData.tags.append(new_tag)
+            return Response(response={}, status=201)
+        return Response(None, status=402)
```

Comment:
402 is a bit strange status code in here. Maybe 400 would be correct in here?

Reply:
That's a brainfart, it was supposed to be 400.